plugin/nabu/src/main/java/ca/on/oicr/gsi/shesmu/nabu/NabuValue.java

```diff
@@ -25,37 +25,37 @@ public NabuValue(long fileswid, String filepath, String qcstatus, String usernam
 		this.qcdate = qcdate;
 	}
 
-	@Export(type = "s")
+	@Export
 	public String comment() {
 		return comment;
 	}
 
-	@Export(type = "s")
+	@Export
 	public String filepath() {
 		return filepath;
 	}
 
-	@Export(type = "i")
+	@Export
 	public long fileswid() {
 		return fileswid;
 	}
 
-	@Export(type = "s")
+	@Export
 	public String project() {
 		return project;
 	}
 
-	@Export(type = "d")
+	@Export
 	public Instant qcdate() {
 		return qcdate;
 	}
 
-	@Export(type = "s")
+	@Export
 	public String qcstatus() {
 		return qcstatus;
 	}
 
-	@Export(type = "s")
+	@Export
```

Comment:
Should these be changed to `@ShesmuVariable` like in the other files?

Reply:
Yes...I'm not sure why they didn't.